docs/manual.rst

```diff
@@ -5,7 +5,7 @@ The Shapely User Manual
 =======================
 
 :Author: Sean Gillies, <sean.gillies@gmail.com>
-:Version: 1.7.0
+:Version: 2.0b1
```

Comment:
Unrelated to this PR but this number is shown on `https://shapely.readthedocs.io/en/latest/manual.html` and needed to be bumped.

Reply:
That reminded me we can actually let this fill in by sphinx -> https://github.com/shapely/shapely/pull/1595